content/highlights/21/_index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "JavaFX 21 Highlights"
+section: "highlights"
+styleclass: "content"
+---
+# JavaFX 21 Highlights
+
+JavaFX version 21 has been released. We've tailored down some of the most exciting parts of the release in this document.
+
+### JavaFX 21 requires macOS 11 or later
+
+On Mac platforms, JavaFX 21 requires macOS 11 or later.
+An exception will be thrown when initializing the JavaFX runtime on older versions of macOS.
+
+### JavaFX 21 requires GTK 3 on Linux
+
+On Linux platforms, JavaFX 21 requires GTK3 version 3.8 or later.
+An exception will be thrown when initializing the JavaFX runtime if the GTK 3 library cannot be loaded.
+
+Exciting features:
+- New APIs:
+  - Add and remove listeners with ease using the new `Subscription` API
+  - Event handler methods `addEvent*` and `removeEvent*` have been added to `EventTarget` interface
+  - GridPane has a new constructor: `GridPane(double hgap, double vgap)`
+  - Platform class has a new method `canStartNestedEventLoop` which can used to check whether it is safe to start a nested event loop
+  - TextFlow has a new method `underlineShape(int, int)` to enable spellchecker decoration (squiggly line) under text region
+  - TableHeaderRow has a new protected method `showColumnMenu(mouseEvent)`, allowing developers to customize the behaviour when the table menu button is clicked
+
```

Comment:
Should we add one line here saying:

```
All new APIs since JavaFX 9 are now listed in the javadoc: https://openjfx.io/javadoc/21/new-list.html
```